static/js/components/UserMenu_test.js

```diff
@@ -9,13 +9,29 @@ import { makeUser } from "../factories/user"
 
 describe("UserMenu component", () => {
   const user = makeUser()
+  it("have MixedLinks for login/registration for mobile view", () => {
+    assert.isOk(
+      shallow(<UserMenu currentUser={user} useScreenOverlay={false} />)
+        .find("MixedLink")
+        .exists()
+    )
+  })
+
+  it("have four menu items with authenticated-menu class attributes", () => {
+    assert.equal(
+      shallow(<UserMenu currentUser={user} useScreenOverlay={true} />)
+        .find(".authenticated-menu")
+        .getElements().length,
```

Comment:
I might be wrong, but I don't think `getElements()` is necessary
```suggestion
       .length,
```